from_cpython/Modules/itertoolsmodule.c

```diff
@@ -4080,6 +4080,30 @@ static PyMethodDef module_methods[] = {
     {NULL,              NULL}           /* sentinel */
 };
 
+PyTypeObject* Itertool_SafeDealloc_Types[] = {
+    // &combinations_type,
+    // &cwr_type,
```

Comment:
definitely add that to the comment (maybe with a `TODO` to take a further look).  confusing otherwise.